src/js/leaflet.storage.xhr.js

```diff
@@ -66,9 +68,10 @@ L.Storage.Xhr = {
                     L.Storage.fire('ui:alert', {content: msg, level: 'error', duration: 100000, actions: actions});
                 }
                 else {
-                    if (xhr.status !== 0) {  // 0 === request cut by user
-                        L.Storage.fire('ui:alert', {'content': L._('Problem in the response'), 'level': 'error'});
-                    }
+                    var errorMessage = L._('HTTP error {errorcode}', {errorcode: xhr.status});
+                    if (xhr.statusText.length > 1)
+                        errorMessage += ": " + xhr.statusText;
```

Comment:
I don't think people want to have a message they can't understand. You may want to output it in the console, though, for advanced users.